src/librustc_trans/trans/cabi_powerpc64le.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2014-2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
The only difference I see is in `classify_ret_ty()`, where the LE version is doing more complicated logic. But I suspect BE should probably also do that logic, in which case the two files would actually be identical?